src/constants/actionParameters.js

```diff
@@ -25,5 +25,18 @@ export const getActionParameters = (option, capability) => {
     }
   }
 
+  if (option === OPTIONS.ITEM_NOTE) {
+    if (capability === CAPABILITIES.ITEM) {
+      return [
+        {
+          key: PARAMETERS_KEYS.STAFF_ONLY,
+          value: false,
+          onlyForActions: [ACTIONS.ADD_TO_EXISTING]
+        },
+      ];
+    }
+  }
```

Comment:
Could you change it to

```suggestion
  if (option === OPTIONS.ITEM_NOTE && capability === CAPABILITIES.ITEM) {
    return [
      {
        key: PARAMETERS_KEYS.STAFF_ONLY,
        value: false,
        onlyForActions: [ACTIONS.ADD_TO_EXISTING]
      },
    ];
  }
```